components/bigquery/query/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Copyright 2018 The Kubeflow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM ml-pipeline-bigquery-base
+
+RUN apt-get update -y && \
```

Comment:
Can you add some comments why we need to install and then purge build-essential? If it is needed can these be moved to base, so keep only the source code and entrypoint in this Dockerfile?

Reply:
I copied this from [the tfdv component](https://github.com/kubeflow/pipelines/blob/master/components/dataflow/tfdv/Dockerfile), so to be honest I'm not sure if it's needed. I will try removing it.

Reply:
Removed.